app/data/models.py

```diff
@@ -34,7 +35,45 @@ class ActionTypes(object):
             (UPDATE, 'Update'),
             (DELETE, 'Delete')
         )
+
         @classmethod
         def as_list(cls):
             return [cls.CREATE, cls.UPDATE, cls.DELETE]
+
     action = models.CharField(max_length=6, choices=ActionTypes.choices)
+
+
+class CeleryJob(models.Model):
```

Comment:
In conversation with Lucien and Kenny, we were talking about needing to track runs of this because it will take forever to look for duplicates in all records so after the first run we'll want to do smart things based on when it was run previously.  So yeah, in this case long-term persistence is useful.

What are the other celery tasks we're planning on?  For exports, I would agree that we don't need extra models on top of what celery provides.  For blackspots, I could see a case for persistence, but maybe there are ways to do it just as well without?